fv_be/app/admin/admin.py

```diff
@@ -70,6 +72,11 @@ class SiteAdmin(BaseAdmin):
 # todo: make suitable admin objects to register for each model
 # todo: make custom admin forms to prevent self selection in ManyToMany fields referring to self
 # ref: https://stackoverflow.com/questions/869856/
-admin.site.register(dictionary_models)
-admin.site.register(Category)
+admin.site.register(DictionaryEntry, DictionaryEntryHiddenBaseAdmin)
```

Comment:
Would it be possible to use the @admin.register decorator in dictionary_admin.py rather than registering in this file?

Reply:
Yes, great point. I added them here as of now because they all have the same baseAdmin class to register. I believe the decorator would come in handy once we start adding/making modifications to individual admin models which can then be registered in the dictionary_admin.py.